app/models/user_viewings.rb

```diff
@@ -0,0 +1,21 @@
+# Contains methods related to User viewings.
+module UserViewings
+  # Returns a list of the user's viewings for the active voting period
+  def current_voting_period_viewings
+    viewings.joins(play: :voting_period).where('active = true')
+  end
+
+  # Returns true if the user has viewed the play and false otherwise
+  def has_viewed? play
+    !viewings.where(play: play).empty?
+  end
+
+  # Takes an array of play ids and creates viewings for these plays using
+  # one SQL statement
+  def insert_viewings play_ids
```

Comment:
`Viewing.create` or even `user.viewings.create` should be able to take an Array of Hashes to create many viewings at once. Can you look into that please? Unless we have to, I'd rather not use `ActiveRecord::Base.connection` directly.